pkg/virt-controller/watch/migration.go

```diff
@@ -1391,12 +1391,14 @@ func (c *MigrationController) getNodeForVMI(vmi *virtv1.VirtualMachineInstance)
 }
 
 func (c *MigrationController) triggerAlertForHostModelUnschedulable(vmi *virtv1.VirtualMachineInstance, targetPod *k8sv1.Pod) {
+	const warningMessage = "Migration cannot proceed since no node is suitable to run the required CPU model / required features"
```

Comment:
Is this even correct? 
We iterate through all nodes but we fire the alert for each node that is not suitable.

Reply:
Thanks for catching that! You're absolutely right!